dynawo/sources/Models/Modelica/Dynawo/Examples/Photovoltaics/WECC/PVCurrentSourceB.mo

```diff
@@ -29,13 +29,16 @@ model PVCurrentSource "WECC PV Model on infinite bus"
     Placement(visible = true, transformation(origin = {80, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 180)));
   Modelica.Blocks.Sources.Constant URefPu(k = 1) annotation(
     Placement(visible = true, transformation(origin = {80, 80}, extent = {{-10, -10}, {10, 10}}, rotation = 180)));
+  Modelica.Blocks.Sources.Constant PFaRef(k = 0.37988) annotation(
```

Comment:
Couldn't we calculate it from PV.P and PV.S?
In case we do change P or S in the test case the reference will be automatically adjusted.

Reply:
Done. I replaced it with acos(PV.PF0), since PF0 is the power factor and is a parameter of the model

Reply:
Right, even a better choice! Thks